projects/cds/src/profiletag/services/profile-tag.injector.spec.ts

```diff
@@ -106,195 +93,108 @@ describe('ProfileTagInjector', () => {
         },
       ],
     });
-
     profileTagInjector = TestBed.get(ProfileTagInjector);
```

Comment:
Should be `profileTagInjector = TestBed.get(ProfileTagInjector as Type<ProfileTagInjector>);`, where `Type` is from `import { Type } from '@angular/core';`